### ISSUE TITLE
Fix GitHub Actions

### ISSUE THREAD

**Issue by DCzajkowski** [DCzajkowski/auth-tests#44]
For whatever reason, GitHub Actions stopped working recently.

Please see this output for debugging purposes: https://github.com/DCzajkowski/auth-tests/pull/42/checks?check_run_id=340654075

It would be much appreciated to get it working again 😊

**Comment by drbyte** [comment]
I'm wondering if Actions has changed `actions/checkout` a bunch in v2 (or whatever `master` is) such that unexpected things are breaking? More investigation could be done in their repo.

In the meantime does this make any difference?
https://github.com/DCzajkowski/auth-tests/blob/7882f4217cce0b541dfa55eafbe07439cdf98393/.github/workflows/run-tests-on-laravel.yml#L14
Force it to `v` via:
```yml
      - uses: actions/checkout@v1
        with:
          fetch-depth: 1
```

**Comment by DCzajkowski** [comment]
Thanks for the idea. Unfortunately it [did not work](https://github.com/DCzajkowski/auth-tests/pull/42/checks?check_run_id=341067784) 😭

**Comment by drbyte** [comment]
The "reference remote is shallow" seems to be the failure-point.

That's where line 17 of the `/actions/setup` script is firing.

It's making reference to the `$commit` var that's set on line 4, and then used as a checkout on line 5, and then lines 16-17 attempt to make composer "install" that repo from the locally-checked-out vcs instance.

https://github.com/DCzajkowski/auth-tests/blob/7882f4217cce0b541dfa55eafbe07439cdf98393/.github/actions/common/setup#L1-L17

I dunno whether a different checkout command on lines 4-5 might make a difference, or whether some other approach to doing a composer-require for the current PR branch could be attempted.

**Comment by DCzajkowski** [comment]
Yeah, currently I am creating a temporary branch with the commit name, because for some reason Composer wants a branch name to be pulled in. The weird thing is it worked for a week or so, but suddenly stopped :/

**Comment by DCzajkowski** [comment]
Fixed in #48. It was a problem with the checkout script. Making it include the entire git history (by setting `fetch-depth: 0`) fixed the problem. Thanks @drbyte for your help!
